NorthstarLauncher/main.cpp

```diff
@@ -1,12 +1,17 @@
 #define WIN32_LEAN_AND_MEAN
-#include <Windows.h>
-#include <TlHelp32.h>
+#include <windows.h>
+#include <tlhelp32.h>
 #include <filesystem>
 #include <sstream>
 #include <fstream>
-#include <Shlwapi.h>
+ #include <shlwapi.h>
```

Comment:
```suggestion
#include <shlwapi.h>
```

Accidental extra space